benchmark/BDN.benchmark/Operations/ObjectOperations.cs

```diff
@@ -27,19 +27,91 @@ public unsafe class ObjectOperations : OperationsBase
         byte[] hSetDelRequestBuffer;
         byte* hSetDelRequestBufferPointer;
 
+        static ReadOnlySpan<byte> HEXISTS => "*3\r\n$7\r\nHEXISTS\r\n$1\r\nf\r\n$1\r\na\r\n"u8;
```

Comment:
This is a lot, and might grow when we add for sorted-set etc. so it might be a good idea to refactor into its own class called `HashObjectOperations` which will hold all the hash operations.

Reply:
Done